p2p/enode/node.go

```diff
@@ -253,3 +253,13 @@ func RandomID(a ID, n int) (b ID) {
 	}
 	return b
 }
+
+// Equal returns a boolean reporting whether a and b contain the same bytes.
+func Equal(a, b ID) bool {
```

Comment:
`ID` is an array type, you can just compare them with `==`. Please remove this function.

Reply:
Done.